www/src/context/SettingsContext.tsx

```diff
@@ -1,21 +1,24 @@
-import React, { createContext, useState, useEffect } from 'react';
+import React, { createContext, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { Helmet } from 'react-helmet';
 import { IntlProvider } from 'react-intl';
-import { messages, type ContainerSize } from '~paragon-react';
+import { messages } from '~paragon-react';
 
-import { THEMES, DEFAULT_THEME } from '../../theme-config';
-import { SETTINGS_EVENTS, sendUserAnalyticsEvent } from '../../segment-events';
+import { useSettings } from '../hooks/useSettings';
+import { useCustomThemes } from '../hooks/useCustomThemes';
+import { useDirection } from '../hooks/useDirection';
+import { useSettingsUI } from '../hooks/useSettingsUI';
 
 export interface IDefaultValue {
   settings: {
-    theme?: string,
     direction?: string,
     language?: string,
-    containerWidth?: ContainerSize,
+    containerWidth?: any,
```

Comment:
Just curious as to why this change is in here, it doesn't feel directly related to the theme picker.